dotCMS/src/integration-test/java/com/dotcms/contenttype/business/RelationshipAPITest.java

```diff
@@ -0,0 +1,87 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.structure.factories.StructureFactory;
+import com.dotmarketing.portlets.structure.model.Relationship;
+import com.dotmarketing.portlets.structure.model.Structure;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RelationshipAPITest extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare () throws Exception {
+
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+
+    }
+
+    /**
+     * Test to Attempt to create a Duplicated Relationship (the system should prevent it)
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void test_create_unique_relationships () throws DotDataException, DotSecurityException {
```

Comment:
Let's name the test according to our _undocumented_ 🤦‍♂️ convention. It would be test<Method_Under_Test>_<StateUnderTest>_<ExpectedBehaviour>. 
In this case it would be something like:
`testSave_RelationshipWithNonUniqueRelationTypeValue_ShouldThrowException`